healthworkforce/mock-services/endpoint.json

```diff
@@ -0,0 +1,103 @@
+{
```

Comment:
@bradsawadye Why is this file duplicated? It already exists in the Importer directory